bin/conf

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# All config is here !!!
+#
+# By Jack @ 2015-03-23
+
+# working directory
+wd=/work/capu/CAPUBBS
```

Comment:
What I am concern is whether user `capu` has permission to access `/work` directly.